integreat_cms/cms/forms/pages/page_form.py

```diff
@@ -44,6 +44,11 @@ class PageForm(CustomModelForm, CustomTreeNodeForm):
         required=False,
         label=_("Enable write access via API for this page"),
     )
+    hix_ignore = forms.BooleanField(
+        required=False,
+        label=_("Ignore HIX value"),
+        help_text=_("Tick if HIX value should not be calculated for this page."),
+    )
```

Comment:
```suggestion
```
You only have to define non-model form fields here. All fields which are derived from the model are automatically created if added to the meta class.

Reply:
Good to know 🙈